Lib/fontParts/base/base.py

```diff
@@ -393,75 +646,132 @@ def pop(self, key, default=None):
             value = self.valueNormalizer.__func__(value)
         return value
 
-    def _pop(self, key, default=None):
-        """
-        Subclasses may override this method.
+    def _pop(self, key: Any, default: Optional[Any]) -> Any:
+        """Remove a key from the native object and return it's value.
+
+        This is the environment implementation of :meth:`BaseDict.pop`.
+
+        :param key: The key to remove. If a :cvar:`BaseDict.keyNormalizer` is
+            set, it will have been applied to the given key.
+        :param default: The default value to return if the key is not found.
+        :return: The value associated with the given key, or the default value
+            if the key is not found. If a :cvar:`BaseDict.valueNormalizer` is set,
+            it will have been applied to the returned value.
```

Comment:
better to say `it will be applied in the calling method to the returned value`

Reply:
Changed.